java/mtsj/core/src/main/java/com/devonfw/application/mtsj/bookingmanagement/logic/impl/BookingmanagementImpl.java

```diff
@@ -239,7 +239,7 @@ public String buildToken(String email, String type) throws NoSuchAlgorithmExcept
     String time = String.format("%02d", ldt1.getHour()) + String.format("%02d", ldt1.getMinute())
         + String.format("%02d", ldt1.getSecond());
 
-    MessageDigest md = MessageDigest.getInstance("SHA-256");
+    MessageDigest md = MessageDigest.getInstance("MD5");
```

Comment:
Any reason why we change from a stronger and more secure hash algorithm to a weak one?

Reply:
in class BookingmanagementImpl.java "MD5" security stuff has been revert back, because the token that it was generating was very long value
(CB_20190228_c06ef778f24d2b616c08261cb3b8566aa34548c7c33fc23f8df19c9f4239d76e) and database field Bookingtoken was only 60, so replaced it with MD5